components/button.html

```diff
@@ -0,0 +1,156 @@
+    <title>Button </title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="/style.css" rel="stylesheet" />
+</head>
+
+<body>
+    <header>
+        <nav class="logo">
+            <ul class="logo-ul">
+                <li><a class="header-color" href="/index.html">HOVER</a></li>
+                <li><span>v1.0</span></li>
+            </ul>
+        </nav>
+        <nav class="menu">
+
+            <ul class="menu-ul ul-main">
+                <li><a href="/index.html">Home</a></li>
+                <li><a href="/components/badge.html">Docs</a></li>
+                <li><a href="https://hover-library.netlify.app/style.css" download>Download</a></li>
+
+                <li><a href="https://github.com/jinath123-yasmin/Hover-Components-CSS-Library">Github</a></li>
+            </ul>
+
+
+        </nav>
+    </header>
+    <div class="main">
+        <aside>
+            <ul>
+
+                <li><a class="ul-main" href="/get_started/introduction.html">Get Started</a></li>
+                <div class="divider"></div>
+                <li><a href="/get_started/introduction.html">Introduction</a></li>
+                <li><a href="/get_started/typography.html">Typography</a></li>
+                <li><a class="ul-main" href="/components/avatar.html">Components</a></li>
+                <div class="divider"></div>
+                <li><a href="/components/avatar.html">Avatar</a></li>
+                <li><a href="/components/badge.html">Badge</a></li>
+                
+                <li><a href="/components/alert.html">Alert</a></li>
+               
+                <li><a href="/components/button.html">Button</a></li>
+                <li><a href="/components/card.html">Card</a></li>
+                <li><a href="/components/Image.html">Image</a></li>
+                <li><a href="/components/Input.html">Input</a></li>
+                <li><a href="/components/List.html">List</a></li>
+                <li><a href="/components/Toast.html">Toast</a></li>
+
+            </ul>
+        </aside>
+        <article>
+            <div class=" section">
+                <h1>Buttons</h1>
+                <p>Buttons are the soul of an interactive Web Application. Buttons allow users to take actions, and make
+                    choices,with a single tap.</p>
+                <div class="divider"></div>
+                <div class="show">
+                    <h1 class="mer">Contained Button</h1>
+                    <div class="btn">
+                        <ul>
+                            <li class="btn_inner pad"> <button class="primary_btn">Primary Button </button></li>
+                            <li class="btn_inner pad"> <button class="secondary_btn">Secondary Button</button></li>
+                            <li class="btn_inner pad"> <button class="tertiary_btn">Tertiary Button</button>
+                            </li>
+                        </ul>
```

Comment:
Your Primary and secondary buttons look similar, how would you use them as your CTA (call to action) while using it in your projects later?  

refer to this and make changes accordingly : 
![image](https://user-images.githubusercontent.com/74822422/117622250-132c2800-b190-11eb-80b4-30093b2943e8.png)

![image](https://user-images.githubusercontent.com/74822422/117622490-4bcc0180-b190-11eb-9539-29634c21a68d.png)

Reply:
Okay! I guess all the buttons have some differences but yeah okay will make changes. Thank You anjali.